components/director/pkg/graphql/schema.graphql

```diff
@@ -206,6 +206,7 @@ enum EventAPISpecType {
 
 type EventAPIDefinition {
     id: ID!
+    applicationID: ID!
```

Comment:
We should either delete `applicationID` from EventAPI and API or add it to documents and webhooks to stay consistent.

Reply:
We decided to keep the `applicationID` field inside the schema, so I added it to documents and webhooks.